src/api/types.ts

```diff
@@ -1594,6 +1600,10 @@ export interface VersionedHostRequirements {
    * Worker node requirements
    */
   worker?: ClusterHostRequirementsDetails;
+  /**
+   * Single node OpenShift node requirements
+   */
+  sno?: ClusterHostRequirementsDetails;
```

Comment:
We can enhance UI for SNO HW reqs.

Reply:
Agree, task here [MGMT-7135](https://issues.redhat.com/browse/MGMT-7135)